modules/oe_content_tender/config/install/field.field.node.oe_tender.oe_documents.yml

```diff
@@ -0,0 +1,32 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.oe_documents
+    - media.type.document
+    - node.type.oe_tender
+  module:
+    - composite_reference
+third_party_settings:
+  composite_reference:
+    composite: false
+id: node.oe_tender.oe_documents
+field_name: oe_documents
+entity_type: node
+bundle: oe_tender
+label: Documents
+description: ''
+required: false
+translatable: true
```

Comment:
Field instances should always be not translatable by default, while storage fields should.

Reply:
fixed